projects/1.18/assets/useful-hats/usefulhats/lang/zh_cn.json

```
@@ -1,4 +1,44 @@
 {
+  "itemGroup.usefulhats": "有用的帽子",
+  "item.usefulhats.aquanaut_helmet": "潜水头盔",
+  "item.usefulhats.aquanaut_helmet.desc.conduit_power": "可获得 %s 秒的 §2潮涌能量§9 效果.",
+  "item.usefulhats.bunny_ears": "兔耳",
+  "item.usefulhats.bunny_ears.desc.jump_boost": "可获得 §2跳跃提升 %s§9 效果.",
+  "item.usefulhats.bunny_ears.desc.eating": "吃东西缩短 §2%s§9 次.",
+  "item.usefulhats.chopping_hat": "木工帽",
+  "item.usefulhats.chopping_hat.desc.chopping_speed": "用斧砍树时加速 §2%s%%§9 时间. (可与附魔及药水效果叠加)",
+  "item.usefulhats.ender_helmet": "末影头盔",
+  "item.usefulhats.ender_helmet.desc.define_teleport": "手持头盔并潜行右键点击以设置传送位置。",
+  "item.usefulhats.ender_helmet.desc.teleport": "使用末影珍珠传送到已设置的位置。",
+  "item.usefulhats.ender_helmet.desc.teleport_position": "位置：X：%s，Y：%s，Z：%s",
+  "item.usefulhats.ender_helmet.message.position_saved": "位置已保存。",
+  "item.usefulhats.ender_helmet.message.position_obstructed": "目标位置被阻挡。",
+  "item.usefulhats.ender_helmet.message.wrong_dimension": "错误的维度。",
+  "item.usefulhats.halo": "光环",
+  "item.usefulhats.halo.desc.no_attack": "除了Boss以外的生物不会攻击你。",
+  "item.usefulhats.halo.desc.beware_of_nether": "除了 §4下界§c 生物。",
+  "item.usefulhats.lucky_hat": "幸运帽",
+  "item.usefulhats.lucky_hat.desc.luck": "握着剑、斧或鱼竿时，可获得 §2幸运%s§9 效果。",
+  "item.usefulhats.lucky_hat.desc.unluck": "但是在钓鱼或杀死生物之后，会获得十秒 §6霉运§c 效果。",
+  "item.usefulhats.mining_hat": "矿工帽",
+  "item.usefulhats.mining_hat.desc.mining_speed": "用镐挖矿时加速 §2%s%%§9 。 (可与附魔及药水效果叠加)",
+  "item.usefulhats.mining_hat.desc.night_vision": "在黑暗中拿着镐时获得 §2夜视§9 效果。",
+  "item.usefulhats.mushroom_hat": "蘑菇帽",
+  "item.usefulhats.mushroom_hat.desc.feeding": "当你饿了，你可以咬一口这顶帽子。",

```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**When you are hungry, you take a bite out of this hat.**
```suggestion
  "item.usefulhats.mushroom_hat.desc.feeding": "饿了你就咬一口这顶帽子。",
```